src/content.js

```diff
@@ -34,7 +35,7 @@ function generateEmptyRoleList() {
   return ul;
 }
 
-function loadProfiles(profile, list, csrf, hidesHistory, hidesAccountId) {
+function loadProfiles(profile, list, csrf, hidesHistory, hidesAccountId, showOnlyMatchingRoles) {
```

Comment:
This addition is wrong? I guess it's not used.

Reply:
Yep, missed that.